typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -685,6 +685,35 @@ export const proteustestnet: ChainMetadata = {
   },
 };
 
+export const mantapacific: ChainMetadata = {
+  chainId: 169,
+  domainId: 169,
+  name: Chains.mantapacific,
+  protocol: ProtocolType.Ethereum,
+  displayName: 'Manta Pacific',
+  displayNameShort: 'Manta',
+  nativeToken: {
+    name: 'Ether',
+    symbol: 'ETH',
+    decimals: 18,
+  },
+  blocks: {
+    confirmations: 1,
+    reorgPeriod: 0,
+    estimateBlockTime: 3,
+  },
+  blockExplorers: [
+    {
+      name: 'Manta Pacific Explorer',
+      url: 'https://pacific-explorer.manta.network/',
```

Comment:
I recommend removing the trailing `/` here in the url. 
The utilities to assemble tx/address urls using this aren't as robust as they should be